packages/envied_generator/lib/src/generate_field.dart

```diff
@@ -17,7 +17,10 @@ Iterable<Field> generateFields(
   bool allowOptional = false,
   bool rawString = false,
 }) {
-  final String type = field.type.getDisplayString(withNullability: false);
+  String type = field.type.getDisplayString();
```

Comment:
Oh, this is to get around the `withNullability` deprecation?

Reply:
Yep.  It's not ideal, but the options were to ignore the deprecated warning (risky since a new stable release may just remove withNullability any day now), make a potentially large breaking change, or just remove the `?` in places that don't yet expect it.

I went with the last option, as it was the fastest and seemed the safest.

Reply:
Yeah, I understand, however, this should be part of another PR, not this one.

Would you be so kind and revert this change inside **this** pull request and make a **new** one addressing this deprecation specifically?

Reply:
I don't mind, but you asked me above to address the linter errors, and these _**are**_ the linter errors.  If not this then, add ignores?

Reply:
Hmm. Revert them here then, I'll then merge this. Then create a PR for the deprecation fix.

Reply:
That's a good question, and I don't know.  But this is the action that failed for my PR with the lint rules:

https://github.com/petercinibulk/envied/actions/runs/12264001806/job/34218460328

And from there:
```
envied_generator:
Analyzing envied_generator...

   info - lib/src/generate_field.dart:20:51 - 'withNullability' is deprecated and shouldn't be used. Only non-nullable by default mode is supported. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
   info - lib/src/generate_field.dart:153:49 - 'withNullability' is deprecated and shouldn't be used. Only non-nullable by default mode is supported. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
   info - lib/src/generate_field_encrypted.dart:24:51 - 'withNullability' is deprecated and shouldn't be used. Only non-nullable by default mode is supported. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
   info - lib/src/generate_field_encrypted.dart:61:51 - 'withNullability' is deprecated and shouldn't be used. Only non-nullable by default mode is supported. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
   info - lib/src/generate_field_encrypted.dart:244:44 - 'withNullability' is deprecated and shouldn't be used. Only non-nullable by default mode is supported. Try replacing the use of the deprecated member with the replacement. - deprecated_member_use
```

Which is why I did those fixes because when you mentioned there were lint issues, those were the ones that the action called out.

Reply:
Hmm, but infos aren't fatal by default.

Could you maybe just revert the `withNullability` changes, so we can try again?

Reply:
Sure.  Should be able to do it in a few hours.